examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Maybe we can align comma separated values in </p><p>Marchi_Re10_u and Marchi_Re10_v</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>